openmetadata-spec/src/main/resources/json/schema/api/data/createTopic.json

```diff
@@ -39,11 +39,6 @@
         "$ref": "../../entity/data/topic.json#/definitions/cleanupPolicy"
       }
     },
-    "sampleData": {
```

Comment:
This has been removed since the `createTopicRequest` doesn't create the sample data. A separate api is available for adding sample data